composer.json

```diff
@@ -59,6 +61,11 @@
     }
   },
   "extra": {
+    "patches": {
+        "friends-of-behat/mink-extension": {
+            "setClient error": "https://github.com/FriendsOfBehat/MinkExtension/files/9480850/16.txt"
```

Comment:
Let's add the PR number `#16` here for easy reference to the issue in case we forget where this came from :)

Reply:
@jhedstrom In my project, this patch failed to apply, I had to add this ignore:

```
"patches-ignore": {
  "drupal/drupal-extension": {
    "friends-of-behat/mink-extension": {
      "setClient error": "https://github.com/FriendsOfBehat/MinkExtension/files/9480850/16.txt"
    }
  }
},
```

`friends-of-behat/mink-extension` 2.3.1 is installed

Reply:
I created https://github.com/mikemadison13/drupalextension/pull/1 (against fork)

Reply:
In my project the patch doesn't apply, friends-of-behat/mink-extension is locked to 2.3.1

Reply:
i wonder if we need to set a minimum version to 2.7.1 of the mink-extension? I'm not sure how we make this PR work without that patch (because it fatal errs out). So we either need to pin way earlier (2.3.1 and remove the patch) or pin later (2.7.1) with. my preference would be the newer version. thoughts @jhedstrom ?